gateway/api/serializers.py

```diff
@@ -43,6 +44,9 @@ class Meta:
     auto_scaling = serializers.BooleanField(
         default=False, required=False, allow_null=True
     )
+    python_version = serializers.CharField(
```

Comment:
Should we use ChoiceField? https://www.django-rest-framework.org/api-guide/fields/#choicefield

Reply:
I realized that the `choicefield` is one of the serializer fields.  These is no choicefield in the model fields. https://docs.djangoproject.com/en/4.2/ref/models/fields/. I guess that it's better leaving it as the charfield.

Reply:
There is a choicefield in model fields and in serializer fields :)

Reply:
Here in models https://docs.djangoproject.com/en/4.2/ref/models/fields/#choices
here in serializers https://www.django-rest-framework.org/api-guide/fields/#choicefield

Reply:
There is a `choices` field **option** that this change is using in the CharField.  But there is no 'ChoiceField' in the Models.  Am I missing?

Reply:
oh, sorry! you are right, I misinterpret that :)

Reply:
There is `CharField` with `choices` option in models (which you are using now) and `ChoicesField` in serializers.
I think `ChoicesField` in serializers is just to check on django side that value passed is correct before sending database query. I think we can can `ChoicesField` here as it is just for verification and value that will be serialized is string, which will be compatible with `CharField` in models.

Reply:
I finally got it.  Thanks!